backend/game/engine/board.py

```diff
@@ -16,28 +16,26 @@ def start_game(self):
         self.players.set_player_order()
         self.STATUS = status.ACTIVE
 
-    def check_if_winner(self, player):
-        if player.victory_points == constants.VICTORY_POINTS_TO_WIN or all(
-                alive_player == player for alive_player
-                in self.players.get_alive_player()):
-            self.WINNER = player
+    def check_if_winner(self, potential_winner):
+        if (potential_winner.victory_points == constants.VICTORY_POINTS_TO_WIN or
+                players.if_last_player_alive(potential_winner)):
+            self.WINNER = potential_winner
             self.end_game()
 
     def end_game(self):
         self.STATUS = status.COMPLETED
```

Comment:
both `self.STATUS` as well as `self.WINNER` should be lower case as they are not object-level constants.